datanode/api/trading_data_v2.go

```diff
@@ -1018,7 +1018,7 @@ func tradesToProto(trades []entities.Trade) []*vega.Trade {
 	return protoTrades
 }
 
-// ListTrades lists trades by using a cursor based pagination model.
+// ListTrades lists trades by.
```

Comment:
```suggestion
// ListTrades lists trades.
```
This is a pretty useless description now though. How about "List all trades" and if there are filters, mentioning that they can be filtered?